bookserver/crud.py

```diff
@@ -404,6 +437,7 @@ async def create_initial_courses_users():
                 downloads_enabled=False,
                 courselevel="",
                 institution="",
+                new_server=False,
```

Comment:
Why this? I don't understand. I thought polls could run on either the bookserver or the web2py server.

Reply:
The new_server variable is only important in the context of web2py where we would be routing the book requests for this course to /ns

Reply:
Still lost, but if it makes sense to you, I'm happy.

Reply:
This was my solution so that we could keep most courses running on the web2py server, but have a few beta testers try running their courses on the new server.  

What do polls have to do with it??

Reply:
I don't know, which is what confuses me. But this is in the `fetch_poll_summary` function, which is called by the `/getpollresults` endpoint, so I assume polls matter for this in some way...

Reply:
OK, my apologies. I look at the code in github completely wrong. Sigh.